boards/arm/nrf52840_pca10059/doc/nrf52840_pca10059.rst

```diff
@@ -135,6 +134,48 @@ Here is an example for the :ref:`blinky-sample` application.
 
 Observe the LED on the board blinking.
 
+Flashing with a device firmware update (DFU)
+============================================
+
+The board is factory-programmed with Nordic's bootloader from Nordic's nRF5 SDK.
+This section covers the steps required to migrate to MCUboot and use its serial
+recovery mode over USB CDC ACM to program new firmware.
+
+Obtain a Nordic-signed MCUboot firmware image, then perform a firmware update
```

Comment:
I'm scared just because of sentence `Obtain a Nordic-signed MCUboot firmware image`. What this mean? Of course I know but i think it should be elaborated more on that, something like:
In order to obtain the mcuboot image you need to build mcuboot for nrf52840_pca10059 target (...) refer to mcuboot documentation. make sure that serial recovery along with usb cdc acm as serial interface was enabled.

Reply:
The bootloader that is factory-programmed on the nrf52840_pca10059 is signed by Nordic and only allows signed bootloader updates; there is no way to update that dongle using the bootloader without an image signed by Nordic.

Reply:
I don't understand you idea.

Reply:
Resolved offline.